axelor-base/src/main/java/com/axelor/apps/base/service/AddressServiceImpl.java

```diff
@@ -270,31 +276,35 @@ public String computeFullName(Address address) {
 
   @Override
   public String computeAddressStr(Address address) {
```

Comment:
we don't need to recompute anything in this method, just return address.getFormattedFullName()